service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -441,24 +439,25 @@ public void testHaveConnectorIntervalsLapsed(final AttributeConnector resourceCo
         request.setAction(ACTION_GET);
         request.setSubjectIdentifier(AGENT_MULDER);
         request.setResourceIdentifier(XFILES_ID);
+        request.setPolicySetsEvaluationOrder(EVALUATION_ORDER_POLICYONE);
         PolicyEvaluationRequestCacheKey key = new PolicyEvaluationRequestCacheKey.Builder().zoneId(ZONE_NAME)
                 .request(request).build();
 
         PolicyEvaluationResult result = mockPermitResult();
         spiedCache.set(key, result);
         spiedCache.get(key);
```

Comment:
@FrankGasparovic, also connectorsNotConfigured() test case should return:

new Object[] { null, null, DateTime.now(), false };